cypress/e2e/auth/a-adminLogIn.cy.js

```diff
@@ -0,0 +1,9 @@
+describe('Admin login', () => {
```

Comment:
Can you name your files in a consistent case. It seems here we are mixing kebab case with camelcase